<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authorization</class>
 <widget class="QWidget" name="Authorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>601</width>
    <height>414</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>269</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose your name:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="names">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>269</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newPlayerBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New player</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="newName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="addPlayerBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delete player</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
